Update the README Description (#27)
greenelab/text_mined_hetnet_manuscript: README.md

<!-- usage note: edit this section. -->

> This repository describes the work done on "reusing label functions to extract multiple types of biomedical relationships from biomedical abstracts at scale". Code attached to this manuscript can be found at [here](https://github.com/greenelab/snorkeling).

Reviewer: ```suggestion
This repository contains the manuscript "Reusing label functions to extract multiple types of biomedical relationships from biomedical abstracts at scale".
Code to produce the results described in this manuscript can be found [here](https://github.com/greenelab/snorkeling).
```
